content/applications/finance/payment_providers/buckaroo.rst

```diff
@@ -5,59 +5,32 @@ Buckaroo
 `Buckaroo <https://www.buckaroo.eu/>`_ is a Dutch-based company that offers several online payment
 possibilities.
 
+.. _payment_providers/buckaroo/configure_dashboard:
+
 Configuration on Buckaroo Plaza
 ===============================
 
-Enable Push responses
----------------------
-
-Log into `Buckaroo Plaza <https://plaza.buckaroo.nl>`_ and go to :menuselection:`My Buckaroo
---> Websites --> Push settings`. Then,
-
-#. Check **Enable Push Response** in the **Delayed and Push responses** section.
-#. | Enter your Odoo database URL followed by `/payment/buckaroo/webhook` in both the **Push URI
-     Success** and **Push URI Failure** text fields.
-   | For example: `https://yourcompany.odoo.com/payment/buckaroo/webhook`.
-#. Leave the other fields at their default value.
-#. Click on **Save** to finalize the configuration.
-
-Configuration on Odoo
+#. Log into `Buckaroo Plaza <https://plaza.buckaroo.nl>`_, go to :menuselection:`My Buckaroo -->
+   Websites` and select the :guilabel:`Push settings` tab.
+#. Select the :guilabel:`Enable Push Response` check box in the :guilabel:`Delayed and Push responses` section.
+#. Enter the URL of your Odoo database, followed by `/payment/buckaroo/webhook` in both the
+   :guilabel:`Push URI Success` and :guilabel:`Push URI Failure` text fields. For example:
+   `https://yourcompany.odoo.com/payment/buckaroo/webhook`.
+#. Leave the other fields as they are and click :guilabel:`Save`.
+#. Go to :menuselection:`Configuration --> Templates`, copy the website key (i.e., the key used to
+   uniquely identify your website with Buckaroo) and save it for later.
+#. Go to :menuselection:`Security --> Secret key`, enter or :guilabel:`Generate` a
+   :guilabel:`Secret key` and click :guilabel:`Save`. Save the key for later.
```

Comment:
```suggestion
#. Go to :menuselection:`Configuration --> Security --> Secret key`, enter or :guilabel:`Generate` a
   :guilabel:`Secret key` and click :guilabel:`Save`. Save the key for later.
```
If you change the previous section to MyBuckaroo, we're no longer in "Configuration/templates", so you need to add where to find "Security"